### PR TITLE
Shruti Assignment 19 done 40mints

### PR DESCRIPTION
**Pull request by ShrutiDJun20**
outputs: Find all differ index from given arrays
image

o/p - Find max value at differed index.
image
 o/pFind all differ index from given arrays.
image

Find first differ index from given arrays
image

### CODE REVIEW

File: src/shrutiDJun20/CompareIntArray.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		//compare length first
+		int diffIndex=0;
+		if(input1.length==input2.length){
+			for(int i=0;i<input1.length;i++){
+				if(input1[i]!= input2[i]){
+					diffIndex=i;
+					System.out.println("first difference found in given arrays is at index "+ diffIndex);
+					return;
+				}
+			}
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/CompareIntArray.java:21]: Where is the else statement ?

File: src/shrutiDJun20/CompareIntArray.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ Find first differ index from given arrays. 
+arr1: {10,2,9,14,3}
+arr2: {10,2,18,14,3}
+o/p:- Values are not matching at index -> 2 
+ */
+package shrutiDJun20;
+
+public class CompareIntArray {
+
+	void compareArray(int[] input1, int[] input2){
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/CompareIntArray.java:11]: Your method name sounds like you are comparing whole array. Your method name should be findDifferIndexOfGivenArray() like this.

File: src/shrutiDJun20/FindAllDiff.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	void findDiffInArr(int[] arr1, int[] arr2){
+		int temp = 0;
+		if(arr1.length==arr2.length){
+			for(int index=0;index<arr1.length;index++){
+				if(arr1[index]!=arr2[index]){
+					temp =index;
+				System.out.println("Values are not matching at index ->"+temp);
+				}
+			}
+		}
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/FindAllDiff.java:26]: Here is also else statement missing.

File: src/shrutiDJun20/CompareIntArray.java (hunk 1 above)
Comment (KrishnaKTechnocredits) [reply src/shrutiDJun20/CompareIntArray.java:21]: for example, array length is not equal then ? At least tell use that given two arrays are of different length.

File: src/shrutiDJun20/StringArrIndexDiff.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	
+	void findDiffStringIndex(String[] input1, String[] input2){
+		String s1 ="";
+		String s2 ="";
+		if(input1.length==input2.length){
+		for(int i = 0 ; i < input1.length ; i++){
+		//	for(int j = 0 ; j < input2.length ; j++){
+				if(!input1[i].equals(input2[i])){
+					s1 = input1[i];
+					s2 = input2[i];
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/StringArrIndexDiff.java]: Why you have used this s1 and s2 extra variable. You should directly print  input1[i] in syso statement.

File: src/shrutiDJun20/CompareIntArray.java (hunk 1 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20/CompareIntArray.java:21]: Changes done as per your comment. Added else statement

File: src/shrutiDJun20/CompareIntArray.java (hunk 2 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20/CompareIntArray.java:11]: changed method name and added else statement too

File: src/shrutiDJun20/FindAllDiff.java (hunk 3 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20/FindAllDiff.java:26]: added else statement as per comment

File: src/shrutiDJun20/StringArrIndexDiff.java (hunk 4 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20/StringArrIndexDiff.java]: removed extra variables

**Pull request merged**